org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -156,6 +149,28 @@ public void start(BundleContext bundleContext) throws Exception {
 		configureProxy();
 	}
 
+	private void setPreferenceNodeId() {
+		Bundle bundle = Platform.getBundle(JDT_UI_PLUGIN);
+		if (bundle != null && bundle.getState() != Bundle.ACTIVE) {
+			// start the org.eclipse.jdt.ui if it exists
+			try {
+				bundle.start();
```

Comment:
gaaah

Reply:
When running tests in a Tycho build, on macOS, Platform.getBundle("org.eclipse.jdt.ui") is active.

Reply:
Do they fail?

Reply:
not with that fix so far